### PR TITLE
sigal: 2.0 -> 2.1

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/sigal/default.nix [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+  version = "2.1";
   pname   = "sigal";
 
   src = python3Packages.fetchPypi {
     inherit version pname;
-    sha256 = "0ff8hpihbd30xjy155ksfpypjskilqg4zmyavgvpri8jaf1qpv89";
+    sha256 = "1ddjijpdg0hk8xzqsgnxicivscyh9lz9cxikkxi8yr9hw2wgwjnm";
   };
 
+  disabled = pythonAtLeast "3.6.0";
```
Comment (marsam) [inline pkgs/applications/misc/sigal/default.nix:12]: IIUC you don't need to add `pythonAtLeast` to the arguments:
```suggestion
  disabled = python3Packages.pythonAtLeast "3.6.0";
```

**Comment by Infinisil** [comment]
@matthiasbeyer Can you update the PR?

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -6605,7 +6605,9 @@ in
 
   sieve-connect = callPackage ../applications/networking/sieve-connect {};
 
-  sigal = callPackage ../applications/misc/sigal { };
+  sigal = callPackage ../applications/misc/sigal {
+    inherit (python.passthru) pythonAtLeast;
```
Comment (Infinisil) [inline pkgs/top-level/all-packages.nix:6609]: No need for this line now. Also, can you squash the commits already?

File: pkgs/applications/misc/sigal/default.nix [hunk 3]
```diff
@@ -1,14 +1,16 @@
-{ lib, python3Packages, ffmpeg_3 }:
+{ lib, python3Packages, pythonAtLeast, ffmpeg }:
```
Comment (risicle) [inline pkgs/applications/misc/sigal/default.nix:1]: You're still referring to `ffmpeg_3` down in `makeWrapperArgs` which is causing this to fail.

Also you don't need the `pythonAtLeast` arg here anymore.

**Pull request merged**